internal/generate/clusterserviceversion/clusterserviceversion_test.go

```diff
@@ -23,11 +23,10 @@ import (
 	"path/filepath"
 	"regexp"
 
+	"github.com/blang/semver"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	"github.com/onsi/gomega/format"
-
-	"github.com/blang/semver"
```

Comment:
This grouping is intentional.

Reply:
I'm in favor of this change for selfish reasons. I use [`goimportssort`](https://github.com/AanZee/goimportssort), which automatically re-sorts imports into these three groups.